### PR TITLE
(maint) Prep for 1.1.0 release

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

**Comment by justinstoller** [comment]
The previous entries look funny. I'll update them, one sec.

**Comment by justinstoller** [comment]
Okay, looks better!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-##1.0.0
+## 1.1.0
+
+* [TK-442](https://tickets.puppetlabs.com/browse/TK-442) - Updates to dropwizard
```
Comment (camlow325) [inline CHANGELOG.md]: "Updates dropwizard" reads better to me than "Updates to dropwizard" does here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+  from 3.1.2 to 3.2.2. NOTE: This is done by bumping the dependency on clj-parent
+  which also brings in the a new version of tk-jetty9. Java 7 is no longer supported.
```
Comment (camlow325) [inline CHANGELOG.md]: s/in the a/in a

**Comment by justinstoller** [comment]
This is why read through, but didn't consider myself competent to sign off on your documentation PR (;

**Pull request merged**